sdks/python/container/build.gradle

```diff
@@ -60,15 +51,13 @@ golang {
   }
 }
 
-docker {
-  name containerImageName(name: "python")
-  files "./build"
+task buildDocker {
```

Comment:
I change project structure of sdks/python/container.
Previously, build.gradle has a task called `docker` to build an image for py2.
Now I left common tasks needed to create a docker image at this file and moved out `docker` task to a separate file, like other python versions. This makes it easy to retire py2 in future and also can make py2 image only. (Previously, it was not possible.) However, now we cannot use `./gradlew -p sdks/python/container docker` command anymore, we should rename the task. I renamed it to `buildDocker`, so the command will change to  `./gradlew -p sdks/python/container buildDocker`. Is this a good change? do we have a better way to handle this?

Reply:
This change sounds good. 

We could also rename it to just `build` . It would make sense to call it like `./gradlew -p sdks/python/container build`

/cc @yifanzou on gradle related questions.

Reply:
I decided to go with `buildAll`. `build` is already used, so cannot use it here.